resources/views/network/admin/internet_access.blade.php

```diff
@@ -57,7 +41,7 @@
                 if (value) {
                     value = moment(value).format("YYYY. MM. DD. HH:mm");
                 } else {
-                    value = 'Nincs hozzáférés'
+                    value = 'N/A'
```

Comment:
It could be something like 'Unlimited until next kkt period' for those having paid kkt; it would be easier to understand.